aten/src/ATen/native/NNPACK.cpp

```diff
@@ -142,7 +128,7 @@ constexpr int weight_width_dim = 3;
 // Often written as 2 + max_dim (extra dims for batch size and channels)
 // constexpr int max_dim = 3;
 
-std::vector<int64_t> conv_output_size(
+static std::vector<int64_t> conv_output_size(
```

Comment:
nit: I'd do this in a separate PR (as we might want to request pick to 1.3 we should keep this as small as possible).

Reply:
Good point, definitely.